net-im/signal-cli-bin/signal-cli-bin-0.13.1-r1.ebuild

```diff
@@ -17,10 +19,9 @@ LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="~amd64"
 
-DEPEND="
-	|| ( virtual/jdk:21 virtual/jre:21 )
+RDEPEND="
+	>=virtual/jre-21:*
```

Comment:
Why `>=..:*`? We know for a fact that signal-cli-bin will work with java >=22?
I prefer to target the :21 slot and we can wait for upstream to decide if it works with >21

Reply:
> Why `>=..:*`? We know for a fact that signal-cli-bin will work with java >=22? I prefer to target the :21 slot and we can wait for upstream to decide if it works with >21

It's very unlikely Java 22 will be packaged. Next LTS Java will be 25 in Sept or Oct next year. Until then I guess Signall will have several version bumps more.  
Are you sure it needs at least Java 21?  Wouldn't also `>=virtual/jre-1.8:*`work?

Reply:
> Until then I guess Signall will have several version bumps more.

So? This PR is for version 0.13.1, not hypothetical future versions.

> Are you sure it needs at least Java 21? Wouldn't also >=virtual/jre-1.8:*work?

Yes, versions 0.12.x needed java 17, but now 0.13.x needs 21.

Reply:
> Why `>=..:*`? We know for a fact that signal-cli-bin will work with java >=22? I prefer to target the :21 slot and we can wait for upstream to decide if it works with >21

Hello @MrRoy ,
first let me say that none of the 13.1.x ebuilds was committed by me.
I did not inspect the ebuilds itself, because I put this PR here because the startup shell script did NOT what it should. 
The question if >=21 or :21 is the better solution is quite ...... but both is absolutely CORRECT and WORKING.
Better focus on that please.

Reply:
@madmartin maybe you have the PRs confused? [This](https://github.com/gentoo/gentoo/pull/36370) PR is different than [the other one you submitted](https://github.com/gentoo/gentoo/pull/36423)

Reply:
> EDIT: 

@madmartin In the future, please put a new reply instead of editing your original comment as a response to my comment, so I get notified that you responded.

> I see that this is not my PR https://github.com/gentoo/gentoo/pull/36423 so forget the second half sentence about the startup script.

If the rest of your comment stands, then let me reply to it:

>  first let me say that none of the 13.1.x ebuilds was committed by me

Correct, they were submitted by me.

> The question if >=21 or :21 is the better solution is quite .....

Quite what?

> but both is absolutely CORRECT and WORKING.
Better focus on that please.

That is literally the one thing this PR changes, what else is there to focus on?
I would much prefer to have this PR done properly, than just merge because it works. The previous version worked too, so why improve it?

Reply:
> > Why `>=..:*`? We know for a fact that signal-cli-bin will work with java >=22? I prefer to target the :21 slot and we can wait for upstream to decide if it works with >21
> 
> Hello @MrRoy , first let me say that none of the 13.1.x ebuilds was committed by me. I did not inspect the ebuilds itself, because I put this PR here because the startup shell script did NOT what it should. The question if >=21 or :21 is the better solution is quite ...... but both is absolutely CORRECT and WORKING. Better focus on that please. EDIT: I see that this is not my PR #36423 so forget the second half sentence about the startup script.

Tedious discussion. Use either of `virtual/jre:21` or `>=virtual/jre-21:*`. But remove whole DEPEND (or does it need `javac` for installation?). This might even get you rid of the patches.

BTW: Just noticed, the first paragraph in [Java Developer Guide#JDK.2FJRE](https://wiki.gentoo.org/wiki/Java_Developer_Guide#JDK.2FJRE) also seems wrong.  Will discuss this on `#gentoo-java`.

Reply:
@vaukai I changed my PR #36423 by adding your proposed simplification.
You may close this PR.